src/git.rs

```diff
@@ -202,6 +201,35 @@ pub(crate) async fn has_unmerged_paths() -> Result<bool, Error> {
     Ok(!String::from_utf8_lossy(&output.stdout).trim().is_empty())
 }
 
+pub(crate) async fn commit(repo: &Path, msg: &str) -> Result<(), Error> {
+    let mut cmd = git_cmd("git commit")?;
+    cmd.arg("commit")
+        .arg("-m")
+        .arg(msg)
+        .arg("--no-gpg-sign")
+        .current_dir(repo)
+        .env("GIT_AUTHOR_NAME", "pre-commit test")
+        .env("GIT_AUTHOR_EMAIL", "test@example.com")
+        .env("GIT_COMMITTER_NAME", "pre-commit test")
+        .env("GIT_COMMITTER_EMAIL", "test@example.com")
```

Comment:
```suggestion
        .env("GIT_AUTHOR_NAME", "prek test")
        .env("GIT_AUTHOR_EMAIL", "test@example.com")
        .env("GIT_COMMITTER_NAME", "prek")
        .env("GIT_COMMITTER_EMAIL", "test@example.com")
```

I would recommend moving this method into `try_repo` module since it only make sense in the `try_reop` context.